### ISSUE TITLE
Symbols of scatter plot disappears after exporting.  

### ISSUE THREAD

**Issue by sophie-cal2020** [pyqtgraph/pyqtgraph#1209]
### Short description

Was trying to run plotting.py in the examples folder.  And I noticed that the symbols of the scatter plots disappears in the .png after exporting.  I have installed pyqtgraph version 0.11.0.dev0 running on python 3.7.4.  I did a search on past issues and it seems that the problem should have been solved already?

**Comment by j9ac9k** [comment]
By `0.11.0.dev0` do you mean `0.11.rc0` or did you install pyqtgraph by aiming pip at the github repo?

**Comment by j9ac9k** [comment]
Not able to replicate from the current dev build:

![image](https://user-images.githubusercontent.com/646398/82931345-a34f6e00-9f3b-11ea-8cae-5f7bc734ee2f.png)

I installed `0.11.rc0` and could not replicate either

![image](https://user-images.githubusercontent.com/646398/82931556-f2959e80-9f3b-11ea-85f7-c9871aa13cf1.png)

**Comment by sophie-cal2020** [comment]
Hi,

I installed version 0.11.0.dev0.  I just tested it again and realised that the problem only happens if I choose to export only a selected plot (e.g. the scatter plot on the top right).  Exporting the entire scene is fine.

![test](https://user-images.githubusercontent.com/65949208/82968488-03a7e380-a000-11ea-8de4-ba3c8a1013ef.png)


Thanks.

**Comment by sophie-cal2020** [comment]
Just to clarify, i installed by aiming pip at the github repo.  Thanks.

**Comment by j9ac9k** [comment]
Can you tell us more about the system and version of Qt bindings you're using?

**Comment by sophie-cal2020** [comment]
I'm running on Windows 10 with PyQt5, version 5.13.0.  Thanks.

**Comment by dmtvanzanten** [comment]
I'm able to reproduce with pyqtgraph (0.11.0) running Windows 10, Python 3.8 and pyqt 5.9.2.
I also noticed that the markers only disappear for panels that aren't the first panel in a GraphicsLayout. 
For example, after running the code below, and exporting the most left panel (using the ui) I do see the markers, When exporting the middle or most right panel, there are no markers visible. 

```
win = pg.GraphicsLayoutWidget()
layout = win.ci

for i in range(3):
    line = pg.ScatterPlotItem(x[::5], y[::5], brush='ffffff')
    panel = win.addPlot()
    panel.addItem(line)
```

**Comment by dmtvanzanten** [comment]
Since CurvePlotItems do appear well in the export regardless of their position in a GraphicsLayout I compared the two .paint() methods. 
The issue seems related to the .transform() used in the ScatterPlotItem.export() function. For example, when I replace line
https://github.com/pyqtgraph/pyqtgraph/blob/c8fc221e8186711d7be51dac4a7b8edb9f2f784c/pyqtgraph/graphicsItems/ScatterPlotItem.py#L842
with 
`p.translate(pts[0,i] + rec['width']/2 + offset[0], pts[1,i] + rec['width']/2 - offset[1])`
I can get the points visible again (using useCache=False of course) when I use an appropriate offset (which is pass in via the self._exportOpts).

**Comment by j9ac9k** [comment]
Fixed by #1420
